.github/workflows/reassign-reviewer.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: actions-ecosystem/action-regex-match@d50fd2e7a37d0e617aea3d7ada663bd56862b9cc # v2.0.2
         with:
           text: ${{ github.event.comment.body }}
-          regex: '^@modular-magician reassign-reviewer ?@?([a-zA-Z0-9-]+)?$'
+          regex: '@modular-magician reassign-reviewer ?@?([a-zA-Z0-9-]+)?'
```

Comment:
Is there a way we could enforce that we only match individual lines? How are new lines represented in `github.event.comment.body`?

Reply:
If you don't have any objection to requiring standalone lines, I do like that better. :-)

taking a quick scan of https://api.github.com/repos/GoogleCloudPlatform/magic-modules/issues/comments, looks like \r\n or \n - guess it depends on the user OS. Apparently `\r` is also a possible value on some OSes. I've adjusted the regex accordingly.

Reply:
Yeah standalone line is what I'd expect, sole entry in a comment isn't.